packages/capsule/yarn.lock

```diff
@@ -7,61 +7,52 @@
   resolved "https://registry.yarnpkg.com/@adraffy/ens-normalize/-/ens-normalize-1.10.0.tgz#d2a39395c587e092d77cbbc80acf956a54f38bf7"
   integrity sha512-nA9XHtlAkYfJxY7bce8DcN7eKxWWCWkU+1GR9d+U6MbNpfwQp8TI7vqOsBsMcHoT4mBu2kypKoSKnghEzOOq5Q==
```

Comment:
yarn.lock at the package level can be removed

Reply:
@vr2amesh please remove the capsule/yarn.lock file